generators/app/templates/webpack.config.babel.js

```diff
@@ -1,4 +1,13 @@
-export default env => {
-    return require(`./config/${env}.js`)({env: env})
-}
+export default (env) => {
+    // Let's check if the --env has a serve string
+    // this is one way we can activate devserve
+    const serve = env.includes('serve')
```

Comment:
Why do this? 
I already had a mechanism where we can merge and override previous configs.. so, we can have a devserver.js that merges in additional stuff on top of dev.src, we don't need this extra stuff here.

this is inconsistent with the rest of the logic. The logic is "dev" specifies the filename, I don't want a "special case" for server.. special cases are not special, they are bad.